apps/io-services-cms-webapp/src/utils/ingestion/formatter/activation-avro-formatter.ts

```diff
@@ -0,0 +1,38 @@
+import { EventData } from "@azure/event-hubs";
+import * as avro from "avsc";
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+
+import { ServiceActivations as avroActivation } from "../../../generated/avro/dto/ServiceActivations";
+import { EnrichedLegacyActivationCosmosResource } from "../enriched-types/activation-pdv-enriched";
+
+export const buildAvroActivationObject = (
+  activationCosmosRecord: EnrichedLegacyActivationCosmosResource,
+): Omit<avroActivation, "schema" | "subject"> => ({
+  id:
+    activationCosmosRecord.userPDVId +
+    activationCosmosRecord.serviceId +
+    activationCosmosRecord._ts,
```

Comment:
question: Do we want to concatenate them with a separator?

Such as `<userPDVId>#<serviceId>#<timestamp>`